netmaster/mastercfg/policyState.go

```diff
@@ -293,6 +293,13 @@ func (gp *EpgPolicy) createOfnetRule(rule *contivModel.Rule, dir string) (*ofnet
 		return nil, err
 	}
 
+	// Send AddRule to netplugin agents
+	err = addPolicyRuleState(ofnetRule)
```

Comment:
In this particular case, remoteEpgID has a function scope and using ':' for err will cause remoteEpgID also to be block scoped and hence I used the var. I agree with your suggestion for boolean checks (esp for map access and the processStateEvent example), but for explicit err value checks we should not enforce this rule. For code readability I feel its better to split it into multiple lines.